src/Symfony/Bridge/Twig/Extension/FormExtension.php

```diff
@@ -23,7 +23,10 @@
  */
 class FormExtension extends \Twig_Extension implements \Twig_Extension_InitRuntimeInterface
 {
-    private $renderer;
+    /**
+     * Make this property private in 4.0.
+     */
+    public $renderer;
```

Comment:
I like the idea but how to avoid access to properties other than `$renderer` in a nice way? Throwing a `RuntimeException`?